jerry-debugger/jerry-client-ws.py

```diff
@@ -729,6 +737,48 @@ def parse_source(debugger, data):
         logging.debug("No pending breakpoints")
 
 
+def src_check_args(debugger, is_src_cmd, args):
+    if args:
+        try:
+            line_num = int(args)
+            if line_num < 0:
+                print("Error: Non-negative integer number expected")
+                return -1
+
+            return line_num
+        except ValueError as val_errno:
+            print("Error: Non-negative integer number expected: %s" % (val_errno))
+            return -1
+    elif is_src_cmd:
+        return debugger.default_viewrange
+    else:
+        print("Display should be called with an argument")
+        return -1
+
+
+def print_source(debugger, line_num):
+    last_bp = debugger.last_breakpoint_hit
+    if not last_bp:
+        return
+
+    lines = last_bp.function.source
+    if last_bp.function.source_name:
+        print("Source: %s" % (last_bp.function.source_name))
+
+    if line_num == 0:
+        start = 0
+        end = len(last_bp.function.source) - 1
```

Comment:
Do we really need to subtract 1? the range method below will generate numbers in the `[start, end)` range, that is the `end` number will not be reached